systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java

```diff
@@ -174,6 +184,162 @@ void testJSONFormatLogging() {
         assertThat(StUtils.checkLogForJSONFormat(eoPods, "user-operator"), is(true));
     }
 
+    @Test
+    @Order(14)
```

Comment:
Can you please delete this `@Order(14)`? As we have new suite, we don't need to order the tests (because we are not deleting anything)